shared_model/validators/transaction_validator.hpp

```diff
@@ -189,7 +189,8 @@ namespace shared_model {
       ReasonsGroupType operator()(
           const detail::PolymorphicWrapper<interface::SetQuorum> &sq) const {
         ReasonsGroupType reason;
-        reason.first = "SetQuorum";
+        addInvalidCommand(reason, "SetAccountQuorum");
```

Comment:
SetQuorum or SetAccountQuorum?

Command's name is SetQuorum

Reply:
Typo